src/exr-wrap/exr-wrap.js

```diff
@@ -1,24 +1,3359 @@
+          }
+
+          function addOnPostRun(cb) {
+              __ATPOSTRUN__.unshift(cb)
+          }
+
+          function writeArrayToMemory(array, buffer) {
+              HEAP8.set(array, buffer)
+          }
+
+          function writeAsciiToMemory(str, buffer, dontAddNull) {
+              for (var i = 0; i < str.length; ++i) {
+                  HEAP8[buffer++ >> 0] = str.charCodeAt(i)
+              }
+              if (!dontAddNull) HEAP8[buffer >> 0] = 0
+          }
+          var runDependencies = 0;
+          var runDependencyWatcher = null;
+          var dependenciesFulfilled = null;
+
+          function addRunDependency(id) {
+              runDependencies++;
+              if (Module["monitorRunDependencies"]) {
+                  Module["monitorRunDependencies"](runDependencies)
+              }
+          }
+
+          function removeRunDependency(id) {
+              runDependencies--;
+              if (Module["monitorRunDependencies"]) {
+                  Module["monitorRunDependencies"](runDependencies)
+              }
+              if (runDependencies == 0) {
+                  if (runDependencyWatcher !== null) {
+                      clearInterval(runDependencyWatcher);
+                      runDependencyWatcher = null
+                  }
+                  if (dependenciesFulfilled) {
+                      var callback = dependenciesFulfilled;
+                      dependenciesFulfilled = null;
+                      callback()
+                  }
+              }
+          }
+          Module["preloadedImages"] = {};
+          Module["preloadedAudios"] = {};
+          var dataURIPrefix = "data:application";
+
+          function isDataURI(filename) {
+            if (typeof filename === 'string') {
+              return String.prototype.startsWith ? filename.startsWith(dataURIPrefix) : filename.indexOf(dataURIPrefix) === 0
+            }
+            return false;
+          }
+
+          function integrateWasmJS() {
+              var wasmTextFile = "exr-wrap.wast";
+              // Note that we always want to require here and not let the fallback be to load via XMLHttpRequest if it's text.
+              // The issue here is that in a parent npm project that requires jeri, if this exr-wrap.js and below requires isn't pre-processed
+              // with webpack, then the parent project doesn't know what to do with it, or we require the parent project to deal with it,
+              // which really isn't an option, therefore require the wasm and handle it here.
+              var wasmBinaryFile = require("./exr-wrap.wasm");
+              var asmjsCodeFile = "exr-wrap.temp.asm.js";
+              if (!isDataURI(wasmTextFile)) {
+                  wasmTextFile = locateFile(wasmTextFile)
+              }
+              if (!isDataURI(wasmBinaryFile) && typeof wasmBinaryFile !== "function") {
+                  wasmBinaryFile = locateFile(wasmBinaryFile)
+              }
```

Comment:
Here I added the `!== "function"` so the locateFile wouldn't be called to trigger the XMLHttpRequest